pytorch/src/main/java/org/bytedeco/pytorch/presets/torch_cuda.java

```diff
@@ -75,7 +77,7 @@ public void map(InfoMap infoMap) {
 
             .put(new Info().enumerate().friendly())
             .put(new Info().javaText("import org.bytedeco.pytorch.*;"))
-            .put(new Info().javaText("import org.bytedeco.pytorch.functions.*;"))
+            .put(new Info().javaText("import org.bytedeco.pytorch.cuda.functions.*;"))
```

Comment:
Why do we need a new package for this?

Reply:
If I remember well, all classes in package `functions` will be compiled during a build without cuda. 
And I added a function that needs a class only available during cuda builds: `TraceEntry`.
So builts without cuda would fail if this function is not in a cuda specific package.

Reply:
Last time I tried to understand how the builder determines the list of packages/directories to compile, I failed.
But the thing is that `cuda.functions` is included in a cuda built and is not in a non-cuda built.

Reply:
Where would that be happening?! I don't get how this is supposed to work like this, but not if they are all inside `functions` instead.

Reply:
Oh wait a minute, that's the target package for torch_cuda ... I'm not sure I understand the reason why you want to make things complicated like this, but ok that's why it's working like this